hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1228,17 +1238,57 @@ public void restart() throws IOException {
       // Allow OM to start as Http Server failure is not fatal.
       LOG.error("OM HttpServer failed to start.", ex);
     }
-
     omRpcServer.start();
+
     isOmRpcServerRunning = true;
 
+    startTrashEmptier(configuration);
+
     registerMXBean();
 
     startJVMPauseMonitor();
     setStartTime();
     omState = State.RUNNING;
   }
 
+
+  /**
+   * @param conf
+   * @throws IOException
+   * Starts a Trash Emptier thread that does an fs.trashRoots and performs
```

Comment:
Please follow general guidelines for the javadoc.
1) Begins with function details.
2) Provide `@param` details.
3) Then `@throws` exception cases.
4) Ending with `@return` info.

Reply:
Done